nengo_viz/static/viz_top_toolbar.js

```diff
@@ -20,7 +20,13 @@ VIZ.Toolbar = function(filename) {
     this.toolbar = $('#top_toolbar')[0];
 
     $('#Open_file_button')[0].addEventListener('click', function () {self.file_browser()});
-    $('#Reset_layout_button')[0].addEventListener('click', function () {self.reset_model_layout()});
+    $('#Reset_layout_button')[0].addEventListener('click', function () {
+        VIZ.Modal.title('Are you sure?');
```

Comment:
As a general design principle, dialog boxes like this should be explicit about what you're confirming.  So instead of "Are you sure?", this should be something like "Are you sure you wish to reset this layout, removing all the graphs and resetting the position of all items?"